docker-compose.yml

```diff
@@ -93,6 +94,21 @@ services:
       XDEBUG_CONFIG: client_host=host.docker.internal discover_client_host=0 # Required for MacOS only.
       WP_PHPUNIT__TESTS_CONFIG: local/public/wp-tests-config.php
 
+  elasticsearch:
+    image: docker.elastic.co/elasticsearch/elasticsearch:7.10.2
+    ports:
+      - "9200:9200"
```

Comment:
**question:** Do we need to expose these ports to the Docker host machine? If they are only used by the WP container then it can reach the ports within Docker layer without exposing them here.